source/adapters/native_cpu/device.cpp

```diff
@@ -160,15 +163,22 @@ UR_APIEXPORT ur_result_t UR_APICALL urDeviceGetInfo(ur_device_handle_t hDevice,
   case UR_DEVICE_INFO_PREFERRED_VECTOR_WIDTH_FLOAT:
   case UR_DEVICE_INFO_PREFERRED_VECTOR_WIDTH_DOUBLE:
   case UR_DEVICE_INFO_PREFERRED_VECTOR_WIDTH_HALF:
+  // TODO: How can we query vector width in a platform
+  // independent way?
```

Comment:
As part of future PRs we could have cpu device descriptors with the option to override the individual device properties (below) with environment variables.